ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Notification creation
+	Happy and error paths for notification creation
+
+ Background: Create new notification
+    Given I am on the Search page
+    When I enter 1 into 'SearchParameters_IdFilter'
+    And I click on 'search-button'
+    Then I should be on the Search page
+	When I click on 'create-button'
+    Then I should be on the Patient page
+
+Scenario: Create and submit notification without errors
+    # Patient page
+    When I enter Test into 'Patient_GivenName'
+    And I enter User into 'Patient_FamilyName'
+    And I enter 1 into 'FormattedDob_Day'
+    And I enter 1 into 'FormattedDob_Month'
+    And I enter 1970 into 'FormattedDob_Year'
+    And I select radio value 'sexId-1'
+    And I select 1 for 'Patient_EthnicityId'
+    And I select radio value 'nhs-number-unknown'
+    And I select 1 for 'Patient_CountryId'
+    And I select radio value 'postcode-no'
+    And I click on 'save-button'
+
+    # Episode page
+    Then I should be on the Episode page
+    When I enter 1 into 'FormattedNotificationDate_Day'
+    And I enter 1 into 'FormattedNotificationDate_Month'
+    And I enter 2019 into 'FormattedNotificationDate_Year'
+    And I select TBS0008 for 'Episode_TBServiceCode'
+    # Wait until javascript has populated the hospital dropdown
+    And I wait for 1 second
+    And I select 0ba53cca-6c30-4743-a21e-0664294e64d3 for 'Episode_HospitalId'
+    And I click on 'save-button'
+
+    # Clinical details page
+    Then I should be on the ClinicalDetails page
+    When I check 'NotificationSiteMap_PULMONARY_'
+    When I enter 1 into 'FormattedDiagnosisDate_Day'
+    And I enter 1 into 'FormattedDiagnosisDate_Month'
+    And I enter 2018 into 'FormattedDiagnosisDate_Year'
+    And I click on 'save-button'
+    
+    Then I should be on the ContactTracing page
+    When I click on 'submit-button'
+    Then I should be on the Overview page
+
+Scenario: Create and submit notification without content
+    When I click on 'submit-button'
+    Then I should be on the Patient page
+    And I should see all submission error messages

```

Comment:
If we're going with this approach, do you know if steps can be sectioned out into feature specific files? E.g. error messages for PatientDetails

Reply:
I think it will just look for any steps defined in the Steps directory - not sure I understand how you suggest we could split it? Have a different step file for each error?! I assume not that haha

Reply:
I was trying to ask about something like a `Steps/PatientDetailsSteps.cs` file, so we're not polluting a single place. 
I think you answered the question in your response though -  `Steps directory`